data/migrations/20190117132244_004-recipe-ingredients.js

```diff
@@ -0,0 +1,17 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('recipe-ingredients', function(tbl) {
+    tbl.increments();
+
+    tbl.integer('recipe_id').unsigned().references('id').inTable('recipe');
```

Comment:
This is hard wrapping your head around a table having references to 2 different foreign keys, great job.